src/components/analyses/listing/Listing.js

```diff
@@ -648,11 +640,9 @@ function Listing(props) {
     const getSelectedAnalyses = (analyses) => {
         const items = analyses ? analyses : selected;
         if (items) {
-            return items
-                .map((id) =>
-                    data?.analyses.find((analysis) => analysis.id === id)
-                )
-                .filter((item) => item !== null && item !== undefined);
```

Comment:
Now that I fixed the race condition this `null` filtering is not required anymore 🤷🏽‍♂️